plugins/hydra_optuna_sweeper/setup.py

```diff
@@ -26,10 +26,8 @@
         "Development Status :: 4 - Beta",
     ],
     install_requires=[
-        # TODO(toshihikoyanase): Change version restriction to hydra-core>=1.1 after hydra-core=1.1.0 is released.
         "hydra-core>=1.1.0.dev7",
-        "optuna<2.5.0",
-        "numpy<1.20.0",  # remove once optuna is upgraded to support numpy 1.20
```

Comment:
this is no longer required in the latest optuna release - verified by install optuna in a clean conda env
```
Successfully installed numpy-1.21.1 optuna-2.8.0

```